src/tests/mod.rs

```diff
@@ -38,6 +38,15 @@ fn initialize() {
   });
 }
 
+/// Copies the files under `src` to `dst`.
+/// The copy is NOT recursive.
+/// The files under `src` are copied under `dst`.
+///
+/// # Arguments
+/// * src: Path to the directory to be copied
+/// * dest: Path to destination
+///
+/// This method causes side effect.
```

Comment:
```suggestion
/// This method causes side effects.
```

Probably worth clarifying what side effects? I assume it means the file writes?